zkevm-circuits/src/evm_circuit/execution/copy_to_log.rs

```diff
@@ -68,7 +71,7 @@ impl<F: Field> ExecutionGadget<F> for CopyToLogGadget<F> {
                 cb.tx_log_lookup(
                     tx_id.expr(),
                     TxLogFieldTag::Data,
-                    i.expr(),
+                    data_start_index.expr() + i.expr(),
```

Comment:
I see that in line 90 you do the following constraints for the next step when the copy is not yet finished:
- next src_addr
- next bytes_left
- next src_addr_end

Shouldn't it also constraint that this parameter is the same?:
- next tx_id

And also the next `data_start_index` is?:
- next data_start_index = data_start_index +  MAX_COPY_BYTES

Please, let me know if I missed something.

Reply:
Cool,  added in https://github.com/appliedzkp/zkevm-circuits/pull/510/commits/e592c5e920eae2677e69dae001b66b1f14cee716 and also constrain data start index start with zero in first copy step, thanks for catching !